pkgs/development/python-modules/llfuse/default.nix

```diff
@@ -11,9 +11,22 @@ buildPythonPackage rec {
     sha256 = "1j9fzxpgmb4rxxyl9jcf84zvznhgi3hnh4hg5vb0qaslxkvng8ii";
   };
 
+  patches = [
+    # Assumes that the osxfuse kernel extension is installed to test
+    # against. Due to an implementation bug in the tests, actually
+    # errors while loading the test suite.
+    ./assume-no-osxfuse.patch
+
+    # Tests features that require /proc
+    ./skip-test_sup_groups-darwin.patch
+  ];
+
```

Comment:
Are these bugs upstream? We usally try to only ship NixOS-specific patches in nixpkgs, and `fetchpatch` patches from PRs instead…

Reply:
Upstreamed the patches at https://github.com/python-llfuse/python-llfuse/pull/23, and removed the need for nixpkgs specific packages entirely by using pytest as it should be used.